pkg/volume/volume.go

```diff
@@ -113,20 +109,6 @@ func (v *Operation) Create() (*v1alpha1.CASVolume, error) {
 		return nil, errors.Errorf("failed to create volume: missing volume capacity: %s", v.volume)
 	}
 
-	pvcName := v.volume.Labels[string(v1alpha1.PersistentVolumeClaimKey)]
-	if len(pvcName) == 0 {
```

Comment:
instead of removing, lets handle the case when the request doesn't have this label

Reply:
done